stable/fairwinds-insights/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "15.3"
```

Comment:
Let's hold off on this PR until we update appVersion to 15.4

Reply:
removed